crawl-ref/source/mon-data.h

```diff
@@ -6960,6 +6974,19 @@ DUMMY(MONS_HELL_LORD, '&', COLOUR_UNDEF, "hell lord", TILEP_MONS_PROGRAM_BUG)
     {TILEP_MONS_AGNES}, TILE_ERROR
 },
 
+{
+    MONS_CIRI, 'i', BROWN, "Ciri",
+    M_UNIQUE | M_HERD | M_WARM_BLOOD | M_SPEAKS | M_FEMALE | M_FIGHTER,
+    MR_NO_FLAGS,
+    25, MONS_SPRIGGAN, MONS_SPRIGGAN, MH_HOLY | MH_DEMONIC, 100,
```

Comment:
We currently avoid using mixed holinesses - they sound cool, but complicate code without really doing much in-game. The only current mixed holinesses are demonspawn, and those could probably just be demons without losing anything.

It's not obvious to me what mechanical effects `holy | demonic` holiness provides. My intuition is that she should be either holy or natural, depending on whether you want her to be affected by things that work on 'living beings'.